include/common/proto_type.hpp

```diff
@@ -1,11 +1,10 @@
-#ifndef PROTOTYPE_H
-#define PROTOTYPE_H
+#pragma once
```

Comment:
I think going to `pragma once` is fine for now, to eliminate the current include guards which lack sufficient uniqueness, but I'm in favor of considering moving to uuid-style guards:

```
#ifndef VIAM_CPP_SDK_HEADER_b8b6bfd7-5b15-4865-9061-d54a570de6ae_INCLUDED
#define VIAM_CPP_SDK_HEADER_b8b6bfd7-5b15-4865-9061-d54a570de6ae_INCLUDED
#endif // VIAM_CPP_SDK_HEADER_b8b6bfd7-5b15-4865-9061-d54a570de6ae_INCLUDED
```

Where each file gets a unique uuid. It would be pretty easy to script the conversion from `#pragma once` to that, so I'm fairly happy with unifying around `#pragma once` for now.